docs/build/configure/_index.md

```diff
@@ -227,17 +227,22 @@ Find more information in our [remotes documentation](/build/configure/parts-and-
 
 To automatically run a specified script when the machine boots, configure a _process_.
 
-Start by giving the process a **Name** (`id` in raw JSON) (an identifier of your choice) and clicking **Create Process**.
+Start by giving the process a **Name** (`id` in raw JSON) (an identifier of your choice) and clicking **Create process**.
 
 Then fill in the following fields:
 
 - **Executable** (`name`): The command you want to execute when your machine connects to the server.
 - **Arguments** (`args`): Optional arguments to follow the command.
-- **Working Directory** (`cwd`): Where you want the process to execute.
+- **Working directory** (`cwd`): Where you want the process to execute.
   An optional setting that defaults to the directory where `viam-server` executes.
-
-You can also toggle whether you want errors and other messages to be logged, and whether to execute the command just once or keep running the process indefinitely.
-In raw JSON, these options are represented by `log` (bool) and `one_shot` (bool), respectively.
+- **Logging** (`log`): Toggle logging of errors and other messages on or off (boolean).
+- **Execute once** (`one_shot`): Toggle whether to execute the command just once or keep running the process indefinitely (boolean).
+  - If true, the process executes once at `viam-server` startup.
+    Until the process completes, `viam-server` startup is blocked and the robot appears offline in the [Viam app](https://app.viam.com), so this should only be used for quick processes.
+  - If false, the process continues to run.
```

Comment:
did you get this info from eng or do you know this?

Reply:
Yeah--see the ticket comment, as well as the custom robot dog tutorial